### PR TITLE
Check the skipJsErrors option in the client driver (closes #609)

### LINKED ISSUE #609: Skip js-errors on the page if the skipJsErrors option is set to true

### PR DESCRIPTION
**Pull request by AlexanderMoskovkin**
/cc @inikulin @helen-dikareva

### CODE REVIEW

**Comment by helen-dikareva** [comment]
lgtm

File: src/client/driver/driver.js [hunk 1]
```diff
@@ -58,14 +66,15 @@ const CURRENT_IFRAME_ERROR_CTORS = {
 var hangingPromise = new Promise(noop);
 
 export default class Driver {
-    constructor (testRunId, selectorTimeout, heartbeatUrl, browserStatusUrl) {
+    constructor (testRunId, selectorTimeout, heartbeatUrl, browserStatusUrl, skipJsErrors) {
```
Comment (inikulin) [inline src/client/driver/driver.js]: We have too much args now, maybe `constructor(testRunId, communicationUrls, options)` instead?

**Comment by inikulin** [comment]
\r-

File: src/client/driver/driver.js (hunk 1 above)
Comment (AlexanderMoskovkin) [inline src/client/driver/driver.js]: I'm agree

**Comment by inikulin** [comment]
lgtm

**Comment by helen-dikareva** [comment]
lgtm

**Pull request merged**